pages/Signin.vue

```diff
@@ -0,0 +1,401 @@
+<template>
+  <!DOCTYPE html>
+  <html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+  </head>
+  <body>
+      <div class="content-box">
```

Comment:
You don't have to use html, head and body tags. You can directly start the code from ```<div class="content-box">```. Also remove all the closing tags.

Reply:
In my html body i have created two section one for content and other for image, so if i remove the body tag the page shows error. But i can remove html tag and head tag, but it will be problematic if i remove body tag.

Reply:
@thejaysharma Please take a look